test/acl_event_test.cpp

```diff
@@ -915,6 +915,7 @@ MT_TEST(acl_event_default_config, event_dependency_exec_order) {
 
         // Check separation between dependent events.
         // That is, they actually waited for the prior events.
+        // coverity[COPY_PASTE_ERROR:FALSE]
         CHECK(end_times[i - 1] < submit_times[i]);
```

Comment:
I just tested this, it looks like it does work. 

Although I'm not a big fan of it since it breaks the pattern of checks that we had going on in the test, I do think it's better than adding a comment.